README.md

```diff
@@ -0,0 +1,130 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Final Capstone Backend
+ This is the Final Capstone's backend, it is a rails API made to serve as the database to our Final Capstone's frontend made with React.
+<hr>
+
+## Built With
+![Ruby](https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white) ![Rails](https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white) ![PostgreSQL](https://img.shields.io/badge/PostgreSQL-316192?style=for-the-badge&logo=postgresql&logoColor=white) ![VSCode](https://img.shields.io/badge/VSCode-0078D4?style=for-the-badge&logo=visual%20studio%20code&logoColor=white)
+-  Ruby (programming language)
+-  Rails (framework)
+-  PostgreSQL (database)
+-  Linters: [Rubocop](https://rubocop.org/)
+-  Linters: [Stylent](https://stylelint.io/)
+-  Visual Studio Code (code editor)
+-  RSpec (testings) 
+<hr>
+
+ ## API Documentation
+ ![](./API.JPG)
+
+- [Final Capstone Database](https://bikes-db.onrender.com/api-docs/index.html)
+<hr>
+
+ ## SQL schema
+ ![](./schema.JPG)
+
+<hr>
+
+ ## Kanban board
+ - [Project link](https://github.com/AbrahaKahsay/final-capstone-backend/projects/1)
+ 
+ ![](https://user-images.githubusercontent.com/84423749/203654321-8fa0e497-98f6-407c-835b-d7dc7cec9f42.png)
+
+ Our group remained the same throughout the whole project.
+
+<hr>
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+-  A code editor like Visual Studio Code with Git and Ruby.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+```
+$ ruby --version && irb
+```
+
+### Install
+
+    In the terminal, navigate to where you wish the repo to be located and run these commands.
+
+```
+$ git clone https://github.com/AbrahaKahsay/final-capstone-backend.git
+```
+```
+$ cd final-capstone-backend
+```
+```
+$ bundle install
+```
+```
+$ rails server
+```
```

Comment:
- [ ] Kindly add the required `rails db:create, rails db:migrate and rails db:seed` that your users will need to setup and run your project. 👍

Reply:
You're right, it slipped our minds, thank you for the feedback